lib/index.js

```diff
@@ -462,6 +464,31 @@ exports.timesSeries = function(times, task) {
 	});
 };
 
+exports.memoize = function(promiseFn, hasher) {
+	var callbackFn = constructIterator(promiseFn, 1, true, false);
+	var memoizedFn = async.memoize(callbackFn);
+
```

Comment:
From the async docs:

`The cache of results is exposed as the memo property of the function returned by memoize.`

It doesn't look like this is supported.